box2d/gradle.properties

```diff
@@ -1 +1 @@
-jtranscVersion=0.6.3
+jtranscVersion=0.6.4-SNAPSHOT
```

Comment:
It broken any way, with old code from 0.6.3 this not worked, but i remember, this misstake because i worked with snapshot locally and commit with other changes. I will try better check it in future.